packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -146,6 +146,11 @@ describe('CSFLE / QE', function () {
       if (!serverSatisfies('>= 6.0.0', true)) {
         return this.skip();
       }
+
+      // temporarily disable for latest-alpha because the CSFLE fails until updated
+      if (!serverSatisfies('< 6.x.x', true)) {
```

Comment:
This would currently skip the tests entirely, I think you want something like

```suggestion
      if (!serverSatisfies('<= 6.x.x', true)) {
```